Move ImageValidator and DescriptionValidator validations to IntegrationValidator

Itay4 opened the pull request:
## Status
- [x] Ready

## Description
 Moved `ImageValidator` and `DescriptionValidator` validations from `file_validator` to `integration`

## Must have
- [x] Tests

yaakovi commented on demisto_sdk/commands/common/hook_validations/integration.py:
```diff
@@ -66,15 +75,18 @@ def is_valid_file(self, validate_rn=True):
             self.is_valid_display_name(),
             self.is_all_params_not_hidden(),
+            self.is_valid_image(),
+            self.is_valid_description(beta_integration=False)
```
```suggestion
            self.is_valid_description(beta_integration=False),
```

yaakovi commented on demisto_sdk/commands/common/hook_validations/integration.py:
```diff
@@ -66,15 +75,18 @@ def is_valid_file(self, validate_rn=True):
             self.is_valid_default_arguments(),
             self.is_valid_beta(),
+            self.is_valid_image(),
+            self.is_valid_description(beta_integration=True)
```
```suggestion
            self.is_valid_description(beta_integration=True),
```

yaakovi commented on demisto_sdk/commands/common/hook_validations/integration.py:
```diff
@@ -572,16 +584,28 @@ def is_valid_display_name(self):
+
         Returns:
-            bool. True if there aren't hidden parameters False otherwise.
+            bool: True if description image/logo is valid, False otherwise.
```
```suggestion
            bool: True if integration description is valid, False otherwise.
```

Itay4 commented:
@yaakovi please see my changes:
 - applied your suggestions
 - reverted `is_all_params_not_hidden` func removal 
 - adjusted `test_is_valid_beta_integration` to check  `is_valid_beta` instead of `is_valid_beta_integration` as it validates only display name anyway
